libs/lora-gateway/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lora-gateway
+PKG_VERSION:=5.0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
No, this will clash with packages further down the road.
Use codeload instead like this:
https://github.com/openwrt/packages/blob/master/sound/fdk-aac/Makefile#L15